Jenkinsfile

```diff
@@ -17,6 +17,7 @@
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
 //@Library(value='pipeline-lib@your_branch') _
+@Library(value='pipeline-lib@sre-2932') _
```

Comment:
So it's intentional to merge with this?

Reply:
That also needs to be removed.

Reply:
Maybe I misunderstood this
> DO NOT REMOVE BRANCH AFTER MERGING AS IT IS NEEDED FOR FC FURTHER VALIDATION

@grom72 do you mean to not remove this branch `grom72/sre-2832-fix-scratch-mount`?

Reply:
The FC Jenkins defaults to "@Library(value='pipeline-lib@sre-2932') _" if it not specified.
This fixes a minor issue about properly running groovy code in a sandbox and is what is needed to use any Jenkins other than in Rio Rancho.

The restriction is that pipeline-lib sre-2932 branch cannot be deleted when it is landed, or jobs will stop working on the FC Jenkins until that default is changed to normal.